### PR TITLE
Warning item

### PR DESCRIPTION
**Pull request by momotofu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui-kit",
-  "version": "0.11.72",
+  "version": "0.11.73",
```
Comment (jakerichan) [inline package.json]: Don't bump versions until you are releasing. What if a different PR gets merged and updates the version before yours?

Comment (momotofu) [inline package.json]: Good point.

File: skins/item/color/yellow.ess [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    border-color hsla(49.6, 73.1%, 26.3%, 1)
+    
+&-input:focus + &-label
+  color hsla(49.6, 73.1%, 26.3%, 1)
+  &:after
+    border-color hsla(49.6, 73.1%, 26.3%, 1)
+    
+// doesn't use variable because the placeholder hijacks it for some reason
+&-input
+  color hsla(49.6, 73.1%, 26.3%, 1)
```
Comment (jakerichan) [inline skins/item/color/yellow.ess]: Yeah, I saw the comment, but that's not cool. We need to find out why it's happening and fix it.  Hard colors are no bueno

File: skins/item/color/yellow.ess [hunk 3]
```diff
@@ -0,0 +1,76 @@
+background = __.color.yellow.light
+color hsla(49.6, 73.1%, 26.3%, 1)
```
Comment (jakerichan) [inline skins/item/color/yellow.ess]: hard set colors are no bueno

Comment (momotofu) [inline skins/item/color/yellow.ess]: This color is not in our palette. We'll update this later when we have more time.

File: skins/item/color/yellow.ess [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  &:active
+    background = __.color.yellow.light 
+    color = __.color.white
+    & </-label:after
+      border-color = __.color.black
+
+&.has-no-hover,
+&.has-no-hover:hover,
+&.has-no-hover:active
+  color hsla(49.6, 73.%, 26.3%, 1)
```
Comment (jakerichan) [inline skins/item/color/yellow.ess]: `73.%`?

File: skins/item/color/yellow.ess [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    border-color hsla(49.6, 73.1%, 26.3%, 1)
+    
+// doesn't use variable because the placeholder hijacks it for some reason
+&-input
+  color hsla(49.6, 73.1%, 26.3%, 1)
+  box-shadow = 'inset 0 -1px 0 ' +__.color.black
+  &::placeholder
+    color = __.color.white.clone().clearer(.5)
+  &:hover  
+    box-shadow inset 0 -1px 0 hsla(49.6, 73.1%, 26.3%, 1)
```
Comment (jakerichan) [inline skins/item/color/yellow.ess]: if you're trying to make yellow darker, use `__.color.yellow.dark[er,est]`

Comment (momotofu) [inline skins/item/color/yellow.ess]: This color is brown. We don't have a variable for it yet.

File: skins/item/color/yellow.ess (hunk 3 above)
Comment (jakerichan) [inline skins/item/color/yellow.ess]: we have plenty of time, let's do it right

Comment (jakerichan) [inline skins/item/color/yellow.ess]: Unless this color will only be used here. If so, we should make the color as a variable at the top of the file.

Comment (momotofu) [inline skins/item/color/yellow.ess]: As far as I can see it's only going to be used here.

Comment (jakerichan) [inline skins/item/color/yellow.ess]: Color is in `utils/color.js`
```
import Color from '../../../utils/color.js'
var brown = new Color('hsla(49.6, 73.1%, 26.3%, 1)')
```

File: skins/item/color/yellow.ess [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  &:active
+    background = __.color.yellow.light 
+    color = __.color.white
+    & </-label:after
+      border-color = __.color.black
+
+&.has-no-hover,
+&.has-no-hover:hover,
+&.has-no-hover:active
+  color hsla(49.6, 73.1%, 26.3%, 1)
```
Comment (jakerichan) [inline skins/item/color/yellow.ess]: missed a brown

Comment (mndvns) [inline skins/item/color/yellow.ess]: ur mom is mr brown

**Comment by jakerichan** [comment]
:+1:

**Pull request merged**